docroot/themes/custom/dcd19/css/style.css

```diff
@@ -8062,14 +8062,15 @@ form .form-item.form-type-textfield textarea.form-control, form .form-item.form-
 form .form-item.form-type-textarea textarea.form-control, form .form-item.form-type-password input.form-control,
 form .form-item.form-type-password textarea.form-control, form .form-item.form-type-url input.form-control,
 form .form-item.form-type-url textarea.form-control, form .form-item.form-type-email input.form-control,
-form .form-item.form-type-email textarea.form-control, form .form-item.form-type-entity-autocomplete input.form-control,
+form .form-item.form-type-email textarea.form-control, form .form-item.form-type-number input.form-control,
+form .form-item.form-type-number textarea.form-control, form .form-item.form-type-entity-autocomplete input.form-control,
 form .form-item.form-type-entity-autocomplete textarea.form-control {
+  font-family: Circular Air;
```

Comment:
@vishalchandna1 there is no font name with "Circular Air", please update the font family name as defined in the fonts file https://github.com/vaibhavjain-in/dcd/blob/dcd-2019/docroot/themes/custom/dcd19/assets/sass/_fonts.scss